platform/components/insights/clusters/clusters-sheet.tsx

```diff
@@ -206,6 +211,9 @@ const RunClusters = ({
           scope: formData.scope,
           instruction: formData.instruction,
           nb_clusters: formData.nb_clusters,
```

Comment:
Please add a condition here : if formData.detect_outlier is true, then set this to null